rust/crd/src/lib.rs

```diff
@@ -511,3 +669,327 @@ impl Configuration for JournalNodeConfig {
         Ok(BTreeMap::new())
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::{HdfsCluster, HdfsRole};
+    use stackable_operator::k8s_openapi::{
+        api::core::v1::ResourceRequirements, apimachinery::pkg::api::resource::Quantity,
+    };
+
+    #[test]
+    pub fn test_pvc_rolegroup_from_yaml() {
+        let cr = "
+---
+apiVersion: hdfs.stackable.tech/v1alpha1
+kind: HdfsCluster
+metadata:
+  name: hdfs
+spec:
+  version: 3.2.2
+  zookeeperConfigMapName: hdfs-zk
+  dfsReplication: 1
+  log4j: |-
+    hadoop.root.logger=INFO,console
+  nameNodes:
+    roleGroups:
+      default:
+        selector:
+          matchLabels:
+            kubernetes.io/os: linux
+        replicas: 2
+  dataNodes:
+    roleGroups:
+      default:
+        config:
+          resources:
+            storage:
+              data:
+                capacity: 5Gi
+        selector:
+          matchLabels:
+            kubernetes.io/os: linux
+        replicas: 1
+  journalNodes:
+    roleGroups:
+      default:
+        selector:
+          matchLabels:
+            kubernetes.io/os: linux
+        replicas: 1
+
+        ";
```

Comment:
Is this for a linter or copy paste?
```suggestion
        replicas: 1";
```